src/zarate.jl

```diff
@@ -254,6 +283,17 @@ function assign_coordinates(layout, graph, layer2nodes)
         end
     end
 
+    # force identical y-positions in certain layers
+    for (layerA, layerB) in force_equal_layers  # ordering in layer1 and layer2 must be identical
+        nodesA = sort(layer2nodes[layerA]) # internal ordering in each layer matters
+        nodesB = sort(layer2nodes[layerB])
+        for (nA, nB) in zip(nodesA, nodesB)
+            yA = variable_by_name(m, "y_$nA")
+            yB = variable_by_name(m, "y_$nB")
```

Comment:
I think this is the same as:

```suggestion
            yA = node2y[nA]
            yB = node2y[nB]
```

Reply:
Nice, hadn't seen the `node2y` variable.